.gitignore

```diff
@@ -31,6 +31,7 @@ go.work.sum
 bundle/manifests/*
 kubeconfig
 config/minikube/.secrets
+config/manager/kustomization.yaml
```

Comment:
Can you please drop this commit from this PR. See https://github.com/RedHatInsights/pg-replication-operator/pull/10#discussion_r2031266707

Reply:
ok